### DISCUSSION TITLE
Loading WHAMR with data.WhamRDataset

### DISCUSSION THREAD

**Discussion by triscuitcircuit** [asteroid-team/asteroid#680] [Q&A] [answered]
Hello!

I have purchased WSJ0 and used it to build a version of the WHAMR! dataset (all 447 gigabytes). During the build process, there was no mention of `.json` files along with the audio tracks. 

However, the Asteroid documentation requires `json_dir` as a path to the directory containing the supposed `.json` files for WHAMR!. 

I thought this was a typo (of some sort), but the source code for the Dataset object also requires `.json` files to be in the actual dataset folder.

Where can I find information pertaining to the needed `.json' files? Is there a script that builds this within Asteroid?

**Comment by mpariente** [comment, marked as answer]
Have you looked at the WHAMR recipe [here](https://github.com/asteroid-team/asteroid/blob/master/egs/whamr/TasNet/run.sh)?  The two first stages should solve your problems.

**Comment by triscuitcircuit** [comment]
Ah that's where it was! I built WHAMR manually, so it didn't have the JSON files that Asteroid expected. 

Maybe that script should be pointed out in the docs?
